pkg/models/stats.go

```diff
@@ -89,3 +89,10 @@ type SystemUserCountStats struct {
 type GetSystemUserCountStatsQuery struct {
 	Result *SystemUserCountStats
 }
+
+type ActiveUserCountStats struct {
+	Count int64
+}
```

Comment:
I think for formatting purposes, it's best to separate the two types with a blank line.
```suggestion
}

```